advanced_samples/Bulk_Upload/DotNet/BulkUploader/appsettings.json

```diff
@@ -0,0 +1,14 @@
+﻿{
+  "NamespaceId": "",
```

Comment:
Do we want to do the usual 'ENTER_NAMESPACE_ID' dummy values like the other samples? Not terribly attached to the dummy values but it would be consistent.